tests/imposter/private-api-config.yaml

```diff
@@ -8,47 +8,25 @@ resources:
   - method: POST
     path: /session
     requestBody:
-      jsonPath: $.client_id
-      value: "success"
-    response:
-      statusCode: 201
-      template: true
-      content: |
-        {
-          "session_id": "success",
-          "state": "sT@t3",
-          "redirect_uri": "http://example.net/return"
-        }
-
-  - method: POST
-    path: /session
-    requestBody:
-      jsonPath: $.client_id
-      value: "authorization-error"
-    response:
-      statusCode: 201
-      template: true
-      content: |
-        {
-          "session_id": "authorization-error",
-          "state": "sT@t3",
-          "redirect_uri": "http://example.net/return"
-        }
-
-  - method: POST
-    path: /session
-    requestBody:
-      jsonPath: $.client_id
-      value: "check-error"
+      anyOf:
+        - jsonPath: $.client_id
+          value: "success"
+        - jsonPath: $.client_id
+          value: "authorization-error"
+        - jsonPath: $.client_id
+          value: "check-error"
+        - jsonPath: $.client_id
+          value: "access-denied"
+        - jsonPath: $.client_id
+          value: "abandon-fail"
+    capture:
+      cid:
+        jsonPath: $.client_id
+        store: testStore
     response:
       statusCode: 201
       template: true
-      content: |
-        {
-          "session_id": "check-error",
-          "state": "sT@t3",
-          "redirect_uri": "http://example.net/return"
-        }
+      content: '{"session_id": "${stores.testStore.cid}","state": "sT@t3","liam": "yes","redirect_uri": "http://example.net/return"}'
```

Comment:
Nice refactor! What is the `"liam": "yes"`for?

Reply:
has been removed, was a debug message